tests/core/contracts/conftest.py

```diff
@@ -387,51 +387,55 @@ def ArraysContract(web3, ARRAYS_CONTRACT):
     return web3.eth.contract(**ARRAYS_CONTRACT)
 
 
-CONTRACT_FALLBACK_FUNCTION_SOURCE = """
-contract A {
-    uint data;
-    function A() public payable { data = 0; }
-    function getData() returns (uint r) { return data; }
-    function() { data = 1; }
+CONTRACT_PAYABLE_TESTER_SOURCE = """
+contract PayableTester {
+  bool public wasCalled;
+
+  function doPayableCall() public payable {
+    wasCalled = true;
+  }
```

Comment:
It looks like we're never really testing `doPayableCall` here. That's maybe fine (since it should be covered by other tests), but let's remove it from the contract and ABI, then.

Reply:
yeah, I will remove it.